src/main/java/org/jitsi/jigasi/transcription/Transcriber.java

```diff
@@ -343,7 +343,10 @@ public void updateParticipant(String identifier,
                     String language
                         = translationLanguageExtension.getTranslationLanguage();
 
-                    this.updateParticipantTargetLanguage(identifier, language);
+                    if (!participant.getSourceLanguage().equals(language))
```

Comment:
What is getSourceLanguage() returns null? CAn you add and that check.

Reply:
added the null check